sdks/python/apache_beam/pipeline_test.py

```diff
@@ -185,65 +180,6 @@ def apply(self, pcoll):
         ['a-x', 'b-x', 'c-x'],
         sorted(['a', 'b', 'c'] | 'AddSuffix' >> AddSuffix('-x')))
 
-  def test_cached_pvalues_are_refcounted(self):
```

Comment:
Do we have an equivalent test for DirectRunner (i.e. that not all intermediates are stored at the same time?)